### DISCUSSION TITLE
Suggestions to use Perfect with AWS Batch?

### DISCUSSION THREAD

**Discussion by yihengli** [PrefectHQ/prefect#3183] [General]
Hi, I just recently found out this amazing project. I liked the clean and easy ways to build a workflow management system from a data scientist's perspective. Our team is currently heavily using AWS Batch for submitting our data pipeline and also research related jobs. If there are any good practices that we can take from Perfect?

I found Airflow sort of had their AWS Batch integrated functions like https://airflow.apache.org/docs/stable/_api/airflow/contrib/operators/awsbatch_operator/index.html, not sure if Perfect has something similar or any equivalent hacks.

Thank you

**Comment by joshmeek** [comment]
Hi @yihengli there aren't currently any AWS batch integrations yet implemented in Prefect but we encourage contributions to the [Task Library](https://docs.prefect.io/core/task_library/)! I have heard of some users using AWS batch in their own task code but I don't have any good resources or practices yet.
